src/libraries/Microsoft.Extensions.Primitives/ref/Microsoft.Extensions.Primitives.cs

```diff
@@ -54,22 +54,26 @@ public partial interface IChangeToken
         public System.ReadOnlyMemory<char> AsMemory() { throw null; }
         public System.ReadOnlySpan<char> AsSpan() { throw null; }
         public System.ReadOnlySpan<char> AsSpan(int start) { throw null; }
```

Comment:
Missing metadata seen in implementation though I don't agree w/ those additions.

Reply:
Thank you. That's strange, I thought CI should detect this